packages/transaction-controller/src/constants.ts

```diff
@@ -1,10 +1,14 @@
 export const CHAIN_IDS = {
   MAINNET: '0x1',
   GOERLI: '0x5',
+  BASE: '0x2105',
```

Comment:
Are these strictly needed in this PR or only needed in the Optimism implementation?